sqld/src/main.rs

```diff
@@ -18,6 +18,9 @@ struct Cli {
     /// The address and port the PostgreSQL over WebSocket server listens to.
     #[clap(long, short, env = "SQLD_WS_LISTEN_ADDR")]
     ws_listen_addr: Option<SocketAddr>,
+    /// The address and port the Hrana server listens to.
+    #[clap(long, short = 'l', env = "SQLD_HRANA_LISTEN_ADDR")]
```

Comment:
I'm realizing that most shorthands don't make much sense, but I would keep the `-l` for HTTP, wdyt?

Reply:
To be honest, I sneakily stole the `-l` shorthand in the hope that Hrana will become the default protocol, with HTTP being used only in very limited environments where WebSockets don't make sense :)